src/FINDOLOGIC/Api/Responses/Xml21/Properties/Results.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class Results
+{
+    /** @var int $count */
+    private $count;
+
+    /**
+     * Results constructor.
+     * @param SimpleXMLElement $result
+     */
+    public function __construct($result)
```

Comment:
Please typehint this with `SimpleXMLElement` and drop the PHPDoc as it is not really needed anyway.

Reply:
Typehinted all constructors
I also changed all parameters with name `$request` to `$response` to have it more consistent.